hsclient.py

```diff
@@ -0,0 +1,2441 @@
+from hellosign_sdk.utils import HSRequest, HSException, NoAuthMethod, HSAccessTokenAuth, HSFormat, api_resource, api_resource_list
```

Comment:
@oconnor-sn I think you accidentally added this file. I don't think it should be part of this PR. Can you please remove it.

Reply:
@martinytodorov - Thanks Martin - changes were made to hsclient.py which is why I added this file - please review and advise further.